extension/src/background/messages/oauth.ts

```diff
@@ -22,6 +22,9 @@ const handler: PlasmoMessaging.MessageHandler = async (req, res) => {
   const settings = await getUserDataByUsername(user.login);
   await storage.set("settings", settings);
 
+  const color = await getUserChatColor(accessToken, user.id);
+  await storage.set("color", color);
```

Comment:
Hi, I have a question:  If the user don`t set as default color on twich, what happens?

Reply:
I created a new account and the Twitch API doesn't return anything, just a empty string, but apparently this does not affect the display of the username.
![Peek 2024-08-07 23-36](https://github.com/user-attachments/assets/51a6b63c-51ed-4b90-b015-c923d8da05bd)